### PR TITLE
Use new unittesting actions

### PR DESCRIPTION
**Pull request by rchl**

### CODE REVIEW

**Comment by rchl** [comment]
@randy3k might wanna check out the failure. Looks like a type (.py.py).

**Comment by randy3k** [comment]
You could use the branch `v1` (It is a convention for GitHub actions) for stability.

**Comment by randy3k** [comment]
ai, I forgot that the branch v1 is not yet lsp_utils compatible.

**Comment by rchl** [comment]
no worries, this can wait

**Comment by randy3k** [comment]
I have just pushed the new commits to v1.

The flow could be further simplified without the repository name and package name 😄 . I needed them because I am testing the action in UnitTesting repo.

```
      - uses: actions/checkout@v2
      - uses: SublimeText/UnitTesting/actions/setup@v1
        with:
          extra-packages: |
            sublimelsp/LSP@main
            sublimelsp/LSP-pyright
      - uses: SublimeText/UnitTesting/actions/run-tests@v1
```

**Comment by rchl** [comment]
Looking good now. Nice!

**Comment by randy3k** [comment]
Glad to know that. At least it is worth the effort.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -28,10 +33,13 @@ jobs:
       - uses: actions/checkout@v2
       - uses: SublimeText/UnitTesting/actions/setup@v1
         with:
+          sublime_text_version: ${{ matrix.sublime_text.version }}
```
Comment (randy3k) [inline .github/workflows/main.yml:37]: It should be `sublime-text-version`. Github action inputs typically use `-`.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -14,72 +14,33 @@ on:
...
-          sh github.sh install_package_control
-          sh github.sh copy_tested_package true  # true == overwrite existing package directory
-          sh github.sh run_tests
+          sublime-text-version: ${{ matrix.sublime_text.version }}
+          extra-packages: |
+            sublimelsp/LSP@${{ matrix.sublime_text.lsp_tag }}
+            sublimelsp/LSP-pyright
+      - uses: SublimeText/UnitTesting/actions/run-tests@v1
+        with:
+          sublime-text-version: ${{ matrix.sublime_text.version }}
```
Comment (randy3k) [inline .github/workflows/main.yml:43]: one last nit. `sublime-text-version` is not needed for `run-test`. It is only needed in the setup step.

Comment (rchl) [reply .github/workflows/main.yml]: thanks. also just noticed looking at the output

**Comment by randy3k** [comment]
By the way, I noticed that you were testing against `sublimelsp/LSP@main` for ST4. If you want to test against the released version, you just need to put the prefix down, i.e., `sublimelsp/LSP@4070-`.

**Pull request merged**